quickstarts/deploy-vm/main.go

```diff
@@ -0,0 +1,159 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+
+	"github.com/Azure/azure-sdk-for-go/profiles/2017-03-09/network/mgmt/network"
+	"github.com/Azure/azure-sdk-for-go/profiles/2017-03-09/resources/mgmt/resources"
```

Comment:
same as above, let's use direct versions or "latest" profile

Reply:
Resources is a bit tricky since there are different entities in different API versions (see the [package directory](https://github.com/Azure/azure-sdk-for-go/tree/dev/services/resources/mgmt)). This is one where the profile will really help, but would come at the cost of latest features...

For now ideally you'd change this to `github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2017-05-10/resources`

Thank you!!

Reply:
OK, we need to have an actual discussion about this, because it means the installation page will need to be rewritten. There needs to be a __single recomendation__ for services or profiles. Services do not seem appropriate for quickstarts or sample code because they are more complicated and harder to select based on versioning.